### PR TITLE
Relax input validation of tenant mapping status API

### PR DESCRIPTION
**Pull request by PetarTodorovv**
**Description**
Relax the input validation in such a way that we allow only a `state` to be reported on the async status API

**Related issue(s)**
- https://github.com/kyma-incubator/compass/pull/2712
- https://github.com/kyma-incubator/compass/pull/2722

**Pull Request status**
- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] [N/A] Mocks are regenerated, using the automated script

### CODE REVIEW

**Review by StanislavStefanov — APPROVED** [review]

File: components/external-services-mock/internal/notification/handler.go [hunk 1]
```diff
@@ -276,6 +218,64 @@ func (h *Handler) Delete(writer http.ResponseWriter, r *http.Request) {
 	writer.WriteHeader(http.StatusOK)
 }
 
+func (h *Handler) Async(writer http.ResponseWriter, r *http.Request) {
```
Comment (PetarTodorovv) [inline components/external-services-mock/internal/notification/handler.go:221]: nothing changed in the method. Only moved.

**Comment by PetarTodorovv** [comment]
reset-vm

**Comment by PetarTodorovv** [comment]
reset-vm

**Review by StanislavStefanov — APPROVED** [review]

**Pull request merged**